backend/src/main/java/com/darass/darass/project/dto/ProjectCreateRequest.java

```diff
@@ -12,4 +12,6 @@
 
     @NotNull
     private String name;
+
+    private String content;
```

Comment:
```suggestion
    private String description;
```
프로젝트 상세 설명이니까 content보다는 description은 어떤가요?

Reply:
저도 이 부분 고민했는데 description이 더 좋은 것 같네요 ㅎㅎ 반영해볼게요